app/services/user_change_logger.rb

```diff
@@ -0,0 +1,36 @@
+class UserChangeLogger
```

Comment:
The behaviour is tested as part of user_spec.rb including the redaction which is a private method.